src/models/ClientWithProfit.php

```diff
@@ -43,8 +43,8 @@ public function rules()
                     'rma_price',
                     "rent_price",
                     "rent_charge",
-                    "leasing_price",
-                    "leasing_charge",
+                    "installment_price",
+                    "installment_charge",
```

Comment:
You've added additional fields in `hiapi-legacy/src/lib/billing/charges/ProfitReportQuery.php`, not replaced them